src/test/java/uk/gov/digital/ho/pttg/api/HmrcResourceTimeoutIntegrationTest_exhaustRetriesBeforeTimeout.java

```diff
@@ -88,7 +88,7 @@ public void setup() throws JsonProcessingException {
                 .andRespond(withSuccess());
 
         hmrcAccessCodeMockService
-                .expect(requestTo(containsString("/access")))
+                .expect(requestTo(containsString("/accessx")))
```

Comment:
Why not make the url change include the JIRA eg 

hmrcAccessCodeMockService
                .expect(requestTo(containsString("/url-made-to-break-for-EE-21310")))